src/Sulu/Bundle/MediaBundle/Resources/config/services.xml

```diff
@@ -116,7 +116,7 @@
                  public="true">
             <argument type="service" id="sulu_media.disposition_type.resolver"/>
             <argument type="service" id="sulu.repository.media"/>
-            <argument type="service" id="sulu.content.path_cleaner"/>
+            <argument type="service" id="sulu_route.path_cleanup"/>
```

Comment:
Not 100% happy that media bundle depends on a route bundle service but add it to our ToDo list to rediscuss this, alternative is slug it ourselve or use dependency inversion principle to have atleast an own interface on media bundle side which uses pathcleanup or fallbacks to an own slugger if route bundle is not installed (more theoretical case of clean architecture)